velox/exec/Task.cpp

```diff
@@ -2154,14 +2154,13 @@ folly::dynamic Task::toJson() const {
     obj["plan"] = planFragment_.planNode->toString(true, true);
   }
 
-  folly::dynamic driverObj = folly::dynamic::array;
-  int index = 0;
-  for (auto& driver : drivers_) {
-    if (driver) {
-      driverObj[index++] = driver->toJson();
+  folly::dynamic drivers = folly::dynamic::object;
```

Comment:
I think it's more natural to keep it as array and call `pushBack` to fix it

Reply:
I did that initially, but then remembered that @tanjialiang wanted to preserve the driver ID / index and changed to using object.